conf/.env

```diff
@@ -118,12 +118,12 @@ COOKIE_SECURE=false
 # For instructions, see: https://docs.firefly-iii.org/advanced-installation/email
 # If you use Docker or similar, you can set these variables from a file by appending them with _FILE
 MAIL_MAILER=smtp
-MAIL_HOST=127.0.0.1
-MAIL_PORT=25
+MAIL_HOST=__DOMAIN__
+MAIL_PORT=587
 MAIL_FROM=__EMAIL__
-MAIL_USERNAME=null
-MAIL_PASSWORD=null
-MAIL_ENCRYPTION=null
+MAIL_USERNAME=__EMAIL__
+MAIL_PASSWORD=__PASSWORD__
```

Comment:
where is this password variable coming from? 🤔

Reply:
I found this as example in 

https://github.com/YunoHost-Apps/galene_ynh/blob/master/conf/config.json

and thought that this is a standard variable like `__EMAIL__`.

But I guess I'm wrong about that?